datafusion/core/src/config.rs

```diff
@@ -237,6 +237,15 @@ impl BuiltInConfigs {
                  to reduce the number of rows decoded.",
                 false,
             ),
+            #[cfg(test)]
+            ConfigDefinition::new_bool(
```

Comment:
```suggestion
            // skip_failing_rules default to true in tests so that regressions are caught much sooner
            ConfigDefinition::new_bool(
```